@shared/api/types.ts

```diff
@@ -71,12 +71,7 @@ export interface AssetIcons {
 export type Balances = Types.BalanceMap | null;
 
 /* eslint-disable camelcase */
-export type HorizonOperation = (
-  | Horizon.PaymentOperationResponse
-  | Horizon.PathPaymentOperationResponse
-) & {
-  transaction_attr: Horizon.TransactionResponse;
-};
+export type HorizonOperation = any;
```

Comment:
@acharb Ugh, I'm realizing I did a pretty bad job here. This doesn't take into account other Response types (like `CreateAccountOperationResponse`). When I add other types, TS doesn't do a great job of inferring the correct type when we use our helper methods like `getIsPayment`. I need to rethink this 🤔 

Associated ticket:
https://stellarorg.atlassian.net/browse/WAL-409

Reply:
I think this is fine for now, agreed we should probably take another look at this, along with the Balances types